packages/list/mwc-list-base.ts

```diff
@@ -29,6 +29,16 @@ import {Layoutable, ListItemBase, RequestSelectedDetail} from './mwc-list-item-b
 
 export {ActionDetail, createSetFromIndex, isEventMulti, isIndexSet, MWCListIndex, SelectedDetail} from './mwc-list-foundation';
 
+function debounce(
+    callback: <T>(this: T, ...args: any[]) => void, waitInMS = 50) {
```

Comment:
can we please try to avoid `any` (`unknown` is okay for google style guide) since I doubt the fn defined in this file is going to be used somewhere else, we can be more specific with its implementation.